docs/contributing.md

```diff
@@ -60,7 +60,7 @@ For new features, submit an issue with the `enhancement` label.
 - We use [pytest](https://docs.pytest.org/en/6.2.x/) for testing.
 - Review the `tests` folder for examples of unit tests.  Just copy one of these.
 - New indicators should be tested against manually calculated, proven, accurate results.  It is helpful to include your manual calculations spreadsheet in the appropriate indicator folder when [submitting changes](#submitting-changes).
-- Historical Stock Quotes are automatically added as pytest fixtures.  A `History.xlsx` Excel file is included in the `samples` folder that is an exact copy of what is used in the unit tests.  Use this for your manual calculations to ensure that it is correct.  Do not commit changes to this Excel file.
+- Historical Stock Quotes are automatically added as pytest fixtures.  Multiple `.csv` files are included in the `samples` folder that is an exact copy of what is used in the unit tests.  Use this for your manual calculations to ensure that it is correct.  Do not commit changes to this Excel file.
```

Comment:
This spreadsheet is still needed in this context, for manual calculations, so this should be reverted and the file restored OR alternatively, referenced and hyperlinked to [the spreadsheet](tests/indicators/_common/data) in the underlying Stock.Indicators repository.